packages/core/src/tracing/index.js

```diff
@@ -160,6 +165,14 @@ exports.preInit = function preInit(preliminaryConfig) {
  * @param {CollectorPIDStore} _processIdentityProvider
  */
 exports.init = function init(_config, downstreamConnection, _processIdentityProvider) {
+  if (hasExperimentalLoaderFlag()) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      "Instana no longer supports the '--experimental-loader' flag starting from Node.js 18.19.0. " +
```

Comment:
suggestion: I'd suggest it differently. e.g. "Node.js invented breaking changes in 18.19 & 20....thats why we had to drop the support for experiemntal loader as well.."

Reply:
Upfated